### DISCUSSION TITLE
How to display git ignore file status in file tree

### DISCUSSION THREAD

**Discussion by osenvosem** [nvim-tree/nvim-tree.lua#1964] [Q&A]
Nvim-tree doesn't signify that some files and folders are ignored by git (added to .gitignore).
How to display their status?

**Comment by alex-courtis** [comment]
You can toggle visibility of git ignored files via toggle_git_ignored default `I`.

Ignored files will be indicated via `:help nvim-tree.renderer.icons.glyphs.git`

See `:help nvim-tree.git.ignore` for disabling the ignore filter by default.
